apps/mobile/index.js

```
@@ -25,9 +25,12 @@ import './src/components/Map/utils/mapRequestUserAgent'
 // INITIAL SETUP DONE
 
 import * as Sentry from '@sentry/react-native'
+import {BACKGROUND_NOTIFICATION_HEADLESS_TASK} from '@vexl-next/expo-background-notification-socket'
 import {registerRootComponent} from 'expo'
+import {AppRegistry} from 'react-native'
 import App from './src/App'
 import {detectMmkvDataLoss} from './src/utils/mmkv/detectMmkvDataLoss'
+import {processBackgroundSocketNotification} from './src/utils/notifications/notificationReceivedHandler'
```

ESLint

[error] './src/utils/notifications/notificationReceivedHandler' imported multiple times.

(import/no-duplicates)